ipv6/ipv6.go

```diff
@@ -11,8 +12,19 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gexec"
+	"net"
 )
 
+func isIPv4(ip string) bool {
```

Comment:
Is this only used for the nginx test? If yes, can you try to move it into the `describeIpv6NginxTest` function?